common/src/main/java/dev/isxander/yacl3/gui/controllers/PopupControllerScreen.java

```diff
@@ -0,0 +1,80 @@
+package dev.isxander.yacl3.gui.controllers;
+
+import dev.isxander.yacl3.gui.OptionListWidget;
+import dev.isxander.yacl3.gui.YACLScreen;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.util.Mth;
+
+public class PopupControllerScreen extends Screen {
+    private final YACLScreen backgroundYaclScreen;
+    private final OptionListWidget optionListWidget;
+    private final ControllerPopupWidget controllerPopup;
+    public double initialScrollAmount;
+    public double scrollAmount;
+    public double smoothScrollAmount;
```

Comment:
why is this a class field rather than a local in renderBackground?

Reply:
I had initially set it to a class field because I was referencing the ElementListWidgetExt for this, and it had it as a class field. Doesn't matter now though because it has been removed in favor of the controller setting the popup widget's y 👍